source/Ship.cpp

```diff
@@ -3160,6 +3378,9 @@ double Ship::JumpDriveFuel(double jumpDistance) const
 
 double Ship::JumpFuelMissing() const
 {
+	// Carried ships do not jump so do not need jump fuel.
```

Comment:
I was under the impression that carried ship could have a hyperdrive and thus also get the ability to jump. Is this not the case?

Reply:
Another clarification, with fighters refueling the fleet without this conditional check your fighters will try to refuel other fighters instead of helping the fleet.

Reply:
I tested this in continuous again and looks like fighters do jump when they have a hyperdrive.

However, not including this change would interfere with ship refueling because fighters will try to refuel other fighters.

How should this case be handled?

Reply:
I decided to handle this by ship attribute.  I pushed c1606d32189cb5f210dec654081ab5d0a0d5e8b1 to resolve this feedback.